Pythonwin/pywin/framework/interact.py

```diff
@@ -168,7 +168,7 @@ def ColorizeInteractiveCode(self, cdoc, styleStart, stylePyStart):
                         state = STYLE_INTERACTIVE_EOL
                     else:
                         state = stylePyStart  # Start coloring Python code.
-            elif state in [STYLE_INTERACTIVE_OUTPUT]:
+            elif state in (STYLE_INTERACTIVE_OUTPUT):
```

Comment:
I think this needs a trailing `,`?

Reply:
Good one, Mark. Thanks for pointing that out. Have created a new pull request. Thanks.